pkg/controllers/customcluster_controller.go

```diff
@@ -361,10 +362,14 @@ func (r *CustomClusterController) reconcileProvision(ctx context.Context, custom
 
 	// The provisioning process will be successfully completed if the init worker is finished successfully.
 	if initWorker.Status.Phase == corev1.PodSucceeded {
+		if err := r.fetchProvisionedClusterKubeConfig(ctx, customCluster, customMachine); err != nil {
+			log.Error(err, "failed to fetch provisioned cluster kubeConfig")
```

Comment:
is this log clear enough to know which cluster failed to fetch kubeconfig?

Reply:
yes, the log contains the info of cluster. this is a log example：  "msg"="failed to fetch provisioned cluster kubeConfig" "error"="failed to open remote file: file does not exist" "controller"="customcluster" "controllerGroup"="infrastructure.cluster.x-k8s.io" "controllerKind"="CustomCluster" "customCluster"={"name":"cc-customcluster","namespace":"default"} "name"="cc-customcluster" "namespace"="default" "reconcileID"="36f191b6-ebcc-429e-811b-13beabfddfb9"